contracts/modules/STO/USDTiered/USDTieredSTO.sol

```diff
@@ -245,16 +246,16 @@ contract USDTieredSTO is USDTieredSTOStorage, STO {
             require(_usdTokens[i] != address(0), "Invalid USD token");
             usdTokenEnabled[_usdTokens[i]] = true;
         }
-        emit SetAddresses(wallet, reserveWallet, _usdTokens);
-    }
+        emit SetAddresses(wallet, treasuryWallet, _usdTokens);
```

Comment:
Perhaps should emit SetTreasuryWallet with the new value for the treasury wallet (i.e. default DS value or treasuryWallet if non-zero)?